### ISSUE TITLE
Add Magento 2.1 compatibility

### ISSUE THREAD

**Issue by Yonn-Trimoreau** [owebia/module-shipping-core#6]
Could you please add Magento 2.1 compatibility ?

```
"magento/framework": "100.1.*",
"magento/module-backend": "100.1.*",
"magento/module-catalog": "100.1.*",
"magento/module-catalog-inventory": "100.1.*",
"magento/module-checkout": "100.1.*",
"magento/module-config": "100.1.*",
"magento/module-customer": "100.1.*",
"magento/module-eav": "100.1.*",
"magento/module-quote": "100.1.*",
"magento/module-store": "100.1.*",
"magento/module-variable": "100.1.*",
"magento/magento-composer-installer": "*",
"nikic/php-parser": "2.0.*"
```

And btw, your module is really nice, thank you

**Comment by mattyl** [comment]
This would be really helpful, or instruction how to do this manually -as manually modifying the file doesnt seem to do anything, the module is blocking update to 2.1

**Comment by owebia** [comment]
Fixed with issue #7 , thanks to Yonn-Trimoreau

**Comment by mattyl** [comment]
catalog version is now 101.*


Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - owebia/module-shipping-core dev-master requires magento/module-catalog 100.* -> satisfiable by magento/module-catalog[100.0.7, 100.0.2, 100.0.3, 100.0.4, 100.0.5, 100.0.6].
    - owebia/module-shipping-core dev-master requires magento/module-catalog 100.* -> satisfiable by magento/module-catalog[100.0.7, 100.0.2, 100.0.3, 100.0.4, 100.0.5, 100.0.6].
    - owebia/module-shipping-core 1.0 requires magento/module-catalog 100.* -> satisfiable by magento/module-catalog[100.0.7, 100.0.2, 100.0.3, 100.0.4, 100.0.5, 100.0.6].
    - Can only install one of: magento/module-catalog[101.0.0, 100.0.7].
    - Can only install one of: magento/module-catalog[101.0.0, 100.0.2].
    - Can only install one of: magento/module-catalog[101.0.0, 100.0.3].
    - Can only install one of: magento/module-catalog[101.0.0, 100.0.4].
    - Can only install one of: magento/module-catalog[101.0.0, 100.0.5].
    - Can only install one of: magento/module-catalog[101.0.0, 100.0.6].
    - Can only install one of: magento/module-catalog[101.0.0, 100.0.7].
    - magento/product-community-edition 2.1.0 requires magento/module-catalog 101.0.0 -> satisfiable by magento/module-catalog[101.0.0].
    - Installation request for magento/product-community-edition 2.1.0 -> satisfiable by magento/product-community-edition[2.1.0].
    - Installation request for owebia/module-shipping-core @dev -> satisfiable by owebia/module-shipping-core[1.0, dev-master].
